src/app/components/navigation/navigation.component.html

```diff
@@ -7,6 +7,11 @@
       <span class="navbar-toggler-icon"></span>
     </button>
 
+    <label for="theme-switch" class="icon theme-switch" i18n-title title="Switch light/dark theme">
```

Comment:
I have several concerns about the current theme switching implementation (I do not claim I have a solution for all, though):
* having the checkbox associated to this label living in another component breaks the principles of component (they should be independent of each other), I know it breaks the "checkbox" hack but maybe prefer using event to propagate state from child to parent ;
* I don't like the fact that the label contains both light *and* dark, maybe change dynamically the label or just use "Switch theme";
* I am questioning the semantic of the checkbox, it feels associated to form and data input, as opposed to buttons that are more generic to dynamically change the content / apparent of the page. Also the checkbox semantic is checked / unchecked, with the current label, we don't know what check or uncheck means.;
* (more personal) I think I would prefer if we showed both the light and dark theme and highlight in some way the one in use. In the current implementation I had trouble figuring out if the icon was corresponding to the current theme or to the other one I could switch to. 
* Why keep the theme switch out of the menu when the language drop down is in the menu?
* The tab order is somewhat not intuitive, it is the first element focusable even though it is after the language select.
* The cursor should be a pointer to show that the element is interactive.